src/main/java/com/kaltura/client/KalturaClientBase.java

```diff
@@ -257,27 +257,34 @@ public Element doQueue() throws KalturaApiException {
 
 		if (logger.isEnabled())
 		{
-			logger.debug("JSON: [" + kparams + "]");

```

Comment:
Guy,
All these changes should be done here: https://github.com/kaltura/clients-generator/blob/Lynx-12.11.0/sources/java/src/main/java/com/kaltura/client/KalturaClientBase.java